requirements.txt

```diff
@@ -1,4 +1,4 @@
 django-freeradius>=0.1.0a
-openwisp-users>=0.1.8,<0.2
-openwisp-utils>=0.2,<0.3
+openwisp-users>=0.1.8,< 0.2
+openwisp-utils>=0.2,< 0.3.1
```

Comment:
try to remove this because it's already defined in `django-freeradius` so having it here is redundant